llvm/lib/IR/Verifier.cpp

```diff
@@ -172,6 +172,11 @@ struct VerifierSupport {
     }
   }
 
+  void Write(const DPValue *V) {
```

Comment:
The reason for this to be added here in a mostly unrelated-looking patch is because of the other verifier change below - now that we can lookup DPValue users of a DIAssignID, the verifier should be checking them the same as it does for debug intrinsic users, and as this is the first time DPValue has come up in the verifier, it needs a Write implementation for the purpose of printing errors.